SPECS/libXaw/libXaw.spec

```diff
@@ -2,13 +2,13 @@
 
 Summary:        X Athena Widget Set
 Name:           libXaw
-Version:        1.0.13
-Release:        17%{?dist}
+Version:        1.0.15
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://www.x.org
-Source0:        https://www.x.org/pub/individual/lib/%{name}-%{version}.tar.bz2
+Source0:        https://www.x.org/archive/individual/lib/%{name}-%{version}.tar.gz
```

Comment:
Just wondering, did the tarball file type change or did we pick up a different format this time?

Reply:
the publisher dropped the support for bz2 going forward